bcda/cclf/testutils/cclfUtils.go

```diff
@@ -67,15 +66,20 @@ func ImportCCLFPackage(acoSize, environment string) (err error) {
 		//timeStr := time.Now().Add(time.Minute * time.Duration(i-1)).Format("1504059")
 		archiveName = fmt.Sprintf("T.BCD.A%s.ZCY%s.T%s", acoIDNum, dateStr, "0000000")
 		filename := fmt.Sprintf("T.BCD.A%s.%sY%s.T%s", acoIDNum, file.Name(), dateStr, "0000000")
-		sourceFile_fileName := fmt.Sprintf("%s/%s__%s", sourcedir, file.Name(),filename)
-		fileList = append(fileList, sourceFile_fileName)
+		sourceFilename := fmt.Sprintf("%s/%s__%s", sourcedir, file.Name(), filename)
+		fileList = append(fileList, sourceFilename)
 	}
 
 	newZipFile, err := os.Create(fmt.Sprintf("%s/%s", DestDir, archiveName))
 	if err != nil {
 		return err
 	}
-	defer newZipFile.Close() // #nosec G307
+	defer func() {
```

Comment:
I see an opportunity to be more `DRY` here.  Can we define a helper utility for closing files?  Maybe we can define it [here](https://github.com/CMSgov/bcda-app/blob/master/bcda/utils/common.go).

Reply:
I'll give it a shot!